tests/openwisp2/sample_firmware_upgrader/migrations/0005_update_image_type_identifiers.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
```

Comment:
we usually don't do data migrations in sample apps, we just erase the DB and start from scratch because it's only for testing.